modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerTest.java

```diff
@@ -114,498 +100,17 @@ public void tearDown() throws Exception {
     }
 
     @Test
-    public void testCreateZoneWithAutoAdjust() throws Exception {
```

Comment:
A lot of tests were removed. Aren't they needed anymore (if yes, why?), or they are moved to somewhere else?